src/services/peopleSearch.ts

```diff
@@ -58,6 +58,7 @@ export type SearchResult = SearchResultBase &
   );
 
 export type PeopleSearchQuery = {
+  class_year: string;
```

Comment:
I'm not sure why this is here.  It seems unrelated to what you're doing.  Maybe it's a partial merge from another branch?

Reply:
That's from another merge, it should be removed by the latest commit